packages/mdjs-core/src/mdjsProcess.js

```diff
@@ -54,6 +55,11 @@ const defaultMetaPlugins = [
  */
 async function mdjsProcess(mdjs, { setupUnifiedPlugins = [] } = {}) {
   const parser = unified();
+  if (!rehypePrism) {
+    rehypePrism = (await import('rehype-prism')).default;
+    loadLanguages(['md', 'sh', 'yml']);
+    defaultMetaPlugins.splice(6, 0, { plugin: rehypePrism, options: {} });
```

Comment:
This is tricky and possibly can be turned down a bit now that it's _working_ but seemed to make the most sense as I was getting this working.

Reply:
can you use `plugins-manager` for this?

Reply:
🤔 The docs don't make it immediately clear what the usage for something like this would be. In particular, the following has no reference to the original plugins, so I'm not sure how it would be joined to `defaultMetaPlugins`:

```
addPlugin(json); // Add at the bottom (default)
addPlugin(json, {}, { location: 'top' }); // Add at the top/beginning of the array
addPlugin(json, {}, { location: beep }); // Add after (default) plugin 'beep'
addPlugin(json, {}, { location: beep, how: 'before' }); // Add before plugin 'beep'
```

Seems like the right path, but would appreciate some more guidance.

Reply:
That's what the docs seem to say, but it doesn't work when done that way. 😞